package.json

```diff
@@ -1,10 +1,9 @@
 {
   "private": true,
   "scripts": {
-    "reset": "lerna clean --yes && yarn dlx rimraf node_modules && yarn install && yarn build-packages",
-    "bootstrap:ci": "yarn cache clean && yarn install --immutable",
-    "build-packages": "lerna run build --ignore '*jss-sample-*'",
-    "build-apps": "lerna run build --scope 'jss-sample-*' --npm-client npm",
+    "reset": "lerna clean --yes && yarn dlx rimraf node_modules && yarn install && yarn build",
+    "install:ci": "yarn cache clean --all && yarn install --immutable",
```

Comment:
Noticed we don't actually use this `install:ci` script in `azure-pipelines.yml`. Should we?

Reply:
Did I decide to leave it - for teamcity? do we need it?

Reply:
Oh, I see. You can't use that script when you didn't run `yarn install`

Reply:
I will remove it